estimator/serenity_estimator.cpp

```diff
@@ -19,19 +18,25 @@ class SerenityEstimatorProcess :
     public Process<SerenityEstimatorProcess> {
  public:
   SerenityEstimatorProcess(
-      const lambda::function<Future<ResourceUsage>()>& _usage)
+      const lambda::function<Future<ResourceUsage>()> _usage)
```

Comment:
You can take the function argument by const ref. Just don't store it as a member as const ref. Make sense?